sqlglot/dialects/trino.py

```diff
@@ -30,5 +31,12 @@ class Generator(Presto.Generator):
             exp.JSONPathSubscript,
         }
 
+        TYPE_MAPPINGS = {**Presto.Generator.TYPE_MAPPING, exp.DataType.Type.TDIGEST: "TDIGEST"}
```

Comment:
This type mapping is redundant, `Type.X` maps to `X` by default: https://github.com/tobymao/sqlglot/blob/16691f962822a132e233d61c2b67ec0fc3da51eb/sqlglot/generator.py#L1153